src/openrct2/ride/coaster/meta/AirPoweredVerticalCoaster.h

```diff
@@ -46,5 +46,6 @@ constexpr const RideTypeDescriptor AirPoweredVerticalCoasterRTD =
         { COLOUR_BRIGHT_RED, COLOUR_LIGHT_BLUE, COLOUR_WHITE },
         { COLOUR_LIGHT_ORANGE, COLOUR_BRIGHT_RED, COLOUR_BRIGHT_YELLOW },
     )),
+    SET_FIELD(ColourKey, RideColourKey::Ride)
```

Comment:
Add a trailing comma. It makes for a cleaner diff if we add extra fields later on.
(Applies to all RTDs, of course.)